_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -0,0 +1,256 @@
+---
+id: 0ssw9k
+name: Scrollable element is keyboard accessible
+rule_type: atomic
+description: |
+  This rule checks that scrollable elements can be scrolled by keyboard
+accessibility_requirements:
+  wcag20:2.1.1: # Keyboard (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G202: # Ensuring keyboard control for all functionality
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+Any HTML element that has [visible][] [children][] in the [flat tree][] for which one of the following is true:
+
+- It has a [horizontal scroll distance][scrollable] greater than the [computed][] [left][padding-left] or [right padding][padding-right]; or
+- It has a [vertical scroll distance][scrollable] greater than the [computed][] [top][padding-top] or [bottom padding][padding-bottom]
+
+## Expectation
+
+Each test target is either included in [sequential focus navigation][] or has a [descendant][] in the [flat tree][] that is included in [sequential focus navigation][].
+
```

Comment:
Open question for reviewers: Should I add a note here to explain why this is needed?

Reply:
That seems reasonable 👍